vhdbuilder/packer/init-variables.sh

```diff
@@ -369,6 +369,13 @@ if [ "$OS_TYPE" == "Windows" ]; then
 	fi
 fi
 
+vnet_rg_name=""
+if [[ "${pool_name}" == *nodesigprod* ]]; then
+	vnet_rg_name="nodesigprod-agent-pool"
```

Comment:
this seems to be the right format for windows/ubuntu pool types based on previous PRs? @junjiezhang1997 is this correct?

since those are in different subs I can't just go and check I don't think

Reply:
It's correct. Thanks for the PR!